spec/support/solidus_stripe/checkout_test_helper.rb

```diff
@@ -77,4 +78,146 @@ def find_stripe_iframe
     expect(fieldset).to have_css('iframe') # trigger waiting if the frame is not yet there
     fieldset.find("iframe")
   end
+
+  def clear_stripe_form
+    %w[number expiry cvc postalCode].each do |name|
+      using_wait_time(10) do
+        within_frame(find_stripe_iframe) do
+          field = find(%{input[name="#{name}"]})
+          field.value.length.times { field.send_keys [:backspace] }
```

Comment:
I can chime in on this one, as it's coming from a previous helper. Basically this is the result of trial and error, and was the most stable way of filling in fields in the stripe iframe, I believe partly because of the JavaScript they run whenever a key is typed, but it's just a hypothesis.

Reply:
I can confirm that I have tried several solutions, but this is the most stable one I have found.